content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Security policy examples
+description: Shows common examples of using Istio security policy.
+weight: 60
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+## Background
+
+This page shows common patterns of using Istio security policy. You may find them useful in your deployment or use this
+as a quick reference to example policies.
+
+The policy demonstrated here are just examples and please make necessary changes to adapt to your actual environment
+before applying.
+
+Also read the [authentication tasks](/docs/tasks/security/authentication/authn-policy) and the
+[authorization tasks](/docs/tasks/security/authorization) for hands-on tutorial of using the security policy with much
+more details.
+
+### Require different JWT issuer per host
+
+JWT validation is common on the ingress gateway and you may want to require different JWT issuers for different
+hosts, you can use the authorization policy for fine grained JWT validation in addition to the
+[request authentication](/docs/tasks/security/authentication/authn-policy/#end-user-authentication) policy.
+
+Use the following policy if you want to allow access to the given hosts if JWT principal matches. Access to other hosts
+will always be denied.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: jwt-per-host
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        hosts: ["example.com", "*.example.com"]
+        # the JWT token must have issuer with suffix "@example.com"
+        requestPrincipals: ["*@example.com"]
+    - source:
+        hosts: [".another.org", "*.another.org"]
+        # the JWT token must have issuer with suffix "@another.org"
+        requestPrincipals: ["*@another.org"]
+{{< /text >}}
+
+### Namespace isolation
+
+You want to block all traffic from outside the namespace `foo`, in other words, isolate the namespace `foo` from other
+namespaces. This requires you have already enabled mTLS.
```

Comment:
Shall we instead say "allow access from the same namespace"?

Reply:
I'm just afraid the `same namespace` gives the impression that the policy can be used for any namespace but in fact you need to explicitly specify the `foo` namespace, in other words, we do not have the semantics of `current.namespace`.

Reply:
I mean "block" is a bit misleading because this is ALLOW policy.

Reply:
updated.